distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.web.cfg

```diff
@@ -38,3 +38,15 @@ org.ops4j.pax.web.ssl.keypassword=${javax.net.ssl.keyStorePassword}
 
 # Client authenticate at server is wanted
 org.ops4j.pax.web.ssl.clientauthwanted=true
+
+#Included SSL/TLS Cipher Suites Regular Expressions
+org.ops4j.pax.web.ssl.ciphersuites.included=${https.cipherSuites}
+
+#Excluded SSL/TLS Cipher Suites Regular Expressions
+org.ops4j.pax.web.ssl.ciphersuites.excluded=.*RC4.*,SSL.*,.*anon.*,.*3DES.*
```

Comment:
Can retire triple DES? It looks like Windows XP and Windows Server 2003 do not support AES.  Those OSes do not support TLS 1.1 or TLS 1.2.  I know the OSes are past end-of-life, but the military isn't known for upgrading systems in a timely manner.